sessions/.gitignore

```diff
@@ -0,0 +1 @@
+/vendor
```

Comment:
The gitignores for the other samples include `/.idea` and `.env`, for consistency should we include those here too?

Reply:
Yes, I think these should be included for all laravel/lumen samples!

Reply:
The reason I didn't include these is that this is not a Laravel/Lumen sample. So at least `.env` is not necessary. `/.idea` I could go either way on, but I'll add it!